config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  get 'render/index'
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  resources :splash, only: [:index]
+  resources :groups do 
+    resources :movements
+  end
```

Comment:
- [ ] It would be nice to use "only" and "except" options to add only necessary routes in case maybe there are some routes that are not used. 👍🏻